Competition2018/src/main/java/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -45,10 +48,10 @@ public void setupShiftGearCommand(OperatorInterface oi, ToggleGearCommand shiftG
     @Inject
     public void setupGripperCommands(OperatorInterface oi, GripperDeployDownCommand down, GripperDeployUpCommand up,
             GripperEjectCommand eject, GripperIntakeCommand intake) {
-        oi.operatorGamepad.getifAvailable(3).whileHeld(up);
+        /*oi.operatorGamepad.getifAvailable(3).whileHeld(up);
```

Comment:
Why is this commented out?

Reply:
Needed more buttons for all the new elevator testing - wrist/intake stuff is going to need to be redesigned in order to use fewer buttons. (Maybe a commandgroup that lowers the wrist + intakes, an eject, and a raise wrist?

Reply:
Got an asana task for that, and students are working on it now.